subsys/bluetooth/host/iso.c

```diff
@@ -864,6 +867,40 @@ void bt_iso_cleanup_acl(struct bt_conn *iso)
 	}
 }
 
+static void store_cis_info(const struct bt_hci_evt_le_cis_established *evt,
+			   struct bt_iso_unicast_info *info)
+{
+	struct bt_iso_unicast_tx_info *central = &info->central;
+	struct bt_iso_unicast_tx_info *peripheral = &info->peripheral;
+
+	info->cig_sync_delay = sys_get_le24(evt->cig_sync_delay);
+	info->cis_sync_delay = sys_get_le24(evt->cis_sync_delay);
+	info->interval = sys_le16_to_cpu(evt->interval);
+	info->max_subevent = evt->nse;
+
+	if (evt->c_max_pdu != 0) {
+		central->bn = evt->c_bn;
+		central->phy = evt->c_phy;
+		central->latency = sys_get_le16(evt->c_latency);
```

Comment:
I think these are relevant even if c_max_pdu is 0. For example the phy in use is relevant to know when evaluating the link budget - you might need to increase TX power if a 2 mbit link is used for ACK/NACK, which might not be needed for a 1mbit link. 
I expect a typical unidirectional stream will use 2mbit for upstream (audio data) whereas downstream (ack's) should be 1mbit to have robust acks/nack transfers.

Reply:
Right. I'll just leave the values as they come from the event then :)